packages/app/universal-ts-utils/src/array/uniq.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * A method to remove duplicates in an array of primitive values
+ */
+export const uniq = <T>(arr: T[]): T[] => Array.from(new Set(arr))
```

Comment:
🟢 Nit: No strong opinion but I would call it `unique` or `removeDuplicates` to make it clear

Reply:
Good suggestion. I was living under the impression that `uniq` is an English word. Turns out it's a linux command 😅 so `unique` it is.